File: ocmsdk/src/main/java/com/gigigo/orchextra/core/domain/entities/article/ArticleRichTextElement.java
package com.gigigo.orchextra.core.domain.entities.article;

import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElement;
import java.io.Serializable;

public class ArticleRichTextElement extends ArticleElement implements Serializable {

  private String html;

  public String getHtml() {
    return html;
  }

  public void setHtml(String html) {
    this.html = html;
  }
}

File: ocmsdk/src/main/java/com/gigigo/orchextra/core/domain/entities/article/ArticleRichTextElement.kt
package com.gigigo.orchextra.core.domain.entities.article

import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElement
import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElementRender

class ArticleRichTextElement : ArticleElement<ArticleRichTextElementRender>()

class ArticleRichTextElementRender : ArticleElementRender() {
  var html: String? = null
}
